plugin/storage/cassandra/spanstore/operation_names.go

```diff
@@ -25,23 +26,51 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/cache"
 	"github.com/jaegertracing/jaeger/pkg/cassandra"
 	casMetrics "github.com/jaegertracing/jaeger/pkg/cassandra/metrics"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
 const (
-	insertOperationName = `INSERT INTO operation_names(service_name, operation_name) VALUES (?, ?)`
-	queryOperationNames = `SELECT operation_name FROM operation_names WHERE service_name = ?`
+	// LatestVersion latest version of operation_names table schema, increase the version if your table schema changes require code change
```

Comment:
`LatestVersion latest version` stutters. You can simply drop the words `latest version` and it will read better.

Also, break the comment into two lines, it's too long for no reason, can't even see on GitHub without scrolling.